workspace/__lib__/xod/core/has-error(pulse)/patch.cpp

```diff
@@ -0,0 +1,20 @@
+
+#pragma XOD error_catch enable
+
+struct State {
+};
+
+{{ GENERATED_CODE }}
+
+void evaluate(Context ctx) {
+    if (isInputDirty<input_IN>(ctx) && getError<input_IN>(ctx)) {
+        emitValue<output_OUT>(ctx, true);
+        setTimeout(ctx, 1);
```

Comment:
```suggestion
        setTimeout(ctx, 0);
```

Otherwise, 1000 us magic duration is introduced